src/vivarium_gates_nutrition_optimization/components/transition.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+from vivarium.framework.engine import Builder
+from vivarium_public_health.disease.transition import ProportionTransition
+
+from vivarium_gates_nutrition_optimization.constants import data_keys
+
+
+class ParturitionSelectionTransition(ProportionTransition):
```

Comment:
I'd suggest moving this to `disease.py` so all the new state machine components are in the same place for ease of reading.